src/components/Timer.jsx

```diff
@@ -4,6 +4,8 @@ export default class Timer extends React.Component {
   constructor(props) {
     super(props);
 
```

Comment:
Er du sikker på det? http://www.newmediacampaigns.com/blog/refactoring-react-components-to-es6-classes